README.md

```diff
@@ -178,6 +181,72 @@ swap is initiated successfully, `loopd` will see the process through.
 
 To query in-flight swap statuses, run `loop monitor`.
 
+### Fees explained
+
+The following is an example output of a 0.01 BTC fast (non-batched) Loop Out
+swap from `testnet`:
+
+```bash
+$ loop out --amt 1000000 --fast
+Max swap fees for 1000000 sat Loop Out: 36046 sat
+Fast swap requested.
+CONTINUE SWAP? (y/n), expand fee detail (x): x
+
+Estimated on-chain sweep fee:        149 sat
+Max on-chain sweep fee:              14900 sat
+Max off-chain swap routing fee:      20010 sat
+Max off-chain prepay routing fee:    36 sat
+Max no show penalty (prepay):        1337 sat
+Max swap fee:                        1100 sat
+CONTINUE SWAP? (y/n):
+```
+
+Explanation:
+
+- **Max swap fees for <x> sat Loop Out** (36'046 sat): The absolute maximum in
+  fees that need to be paid. This includes on-chain and off-chain fees. This
+  represents the ceiling or worst-case scenario. The actual fees will likely be
+  lower. This is the sum of `14900 + 20010 + 36 + 1100` (see below).
+- **Estimated on-chain sweep fee** (149 sat): The estimated cost to sweep the
+  HTLC in case of success, calculated based on the _current_ on-chain fees.
+  This value is called `miner_fee` in the gRPC/REST responses.
+- **Max on-chain sweep fee** (14'900 sat): The maximum on-chain fee the daemon
+  is going to allow for sweeping the HTLC in case of success. The factor of
+  `100` to the estimated fee is applied in case the fees spike between the time
+  the swap is initiated and the time the HTLC can be swept.
+- **Max off-chain swap routing fee** (20'010 sat): The maximum off-chain
+  routing fee that the daemon should pay when finding a route to pay the
+  Lightning invoice. This is a hard limit. If no route with a lower or equal fee
+  is found, the payment is aborted. This value is calculated statically based
+  on the swap amount (base fee 10 sat, fee rate of 2%).
+- **Max off-chain prepay routing fee** (36 sat): The maximum off-chain routing
+  fee that the daemon should pay when finding a route to pay the prepay fee.
+  This is a hard limit. If no route with a lower or equal fee is found, the
+  payment is aborted. This value is calculated statically based on the prepay
+  amount (base fee 10 sat, fee rate of 2%).
+- **Max no show penalty (prepay)** (1'337 sat): This is the amount that has to be
+  pre-paid (off-chain) before the server publishes the HTLC on-chain. This is
+  necessary to ensure the server's on-chain fees are paid if the client aborts
+  and never completes the swap _after_ the HTLC has been published on-chain.
+  If the swap completes normally, this amount is counted towards the full swap
+  amount and therefore is actually a pre-payment and not a fee. This value is
+  called `prepay_amt` in the gRPC/REST responses.
+- **Max swap fee** (1'100 sat): The maximum amount of service fees the server is
+  going to charge for executing the swap. The actual fees might be lower if user
```

Comment:
It is really the maximum swap fee proposed by the server that we are going to accept